src/cdk/dialog/dialog-injectors.ts

```diff
@@ -21,12 +28,20 @@ export const DIALOG_DATA = new InjectionToken<any>('DialogData');
 /** Injection token that can be used to provide default options for the dialog module. */
 export const DEFAULT_DIALOG_CONFIG = new InjectionToken<DialogConfig>('DefaultDialogConfig');
 
-/** @docs-private */
+/**
+ * @docs-private
+ * @deprecated No longer used. To be removed.
+ * @breaking-change 18.0.0
```

Comment:
Would this need to be +2 versions, so v19?

Reply:
We aren't on v17 yet though.

Reply:
If we deprecate this in v17, then it would still need to exist in v18 I think? so v19 is the breaking change

Reply:
Changed.